docs/CHANGELOG.rst

```diff
@@ -15,11 +15,13 @@ Added
 * Add Resolwe API to test url configuration
 * Add TransactionBioProcessTestCase to support writing tests for
   processes which use resdk
+* Add ``genialis/chipseq`` docker image
```

Comment:
Depends on whether this will be open source? I think `resolwe` should be used for open source things?